exposed-core/src/main/java/me/weishu/exposed/ExposedBridge.java

```diff
@@ -286,13 +286,13 @@ public static ModuleLoadResult loadModule(final String moduleApkPath, String mod
     }
```

Comment:
```suggestion
    }
```exposed-core/src/main/java/me/weishu/exposed/ExposedBridge.java